### DISCUSSION TITLE
Set value when getting different value on excel ?

### DISCUSSION THREAD

**Discussion by nqt-tech** [ozlerhakan/poiji#237] [Q&A]
I have a field on excel with the content "agree" and "disagree", but when receiving data, it is "true" and "false". Does Poiji support this part? If yes, how do I install it? Does anyone have an example that I can refer to. Thank you very much.

**Comment by ozlerhakan** [comment]
Hi @nqt-tech ,

You can create a https://github.com/ozlerhakan/poiji#create-custom-formatting for your case.
